pkg/network.go

```diff
@@ -116,6 +116,10 @@ const (
 	// constructing the Knative Route's tag names.
 	DefaultTagTemplate = "{{.Tag}}-{{.Name}}"
 
+	// AutocreateClusterDomainClaimsKey is the key for the
+	// AutocreateClusterDomainClaims property.
+	AutocreateClusterDomainClaimsKey = "autocreateClusterDomainClaims"
```

Comment:
I don't like this name, but I don't have a better suggestion. 🤔

Reply:
same :(. If we can come up with something better before we make this "beta", that'd be nice.